generators/2.0/java/index.js

```diff
@@ -10,6 +11,8 @@ const BASE_RESOURCES = 'src/main/resources';
 const BDK_VERSION_DEFAULT = '1.2.1.BETA';
 const MAVEN_SEARCH_BASE = 'https://search.maven.org/solrsearch/select?q=g:com.symphony.platformsolutions+AND+a:';
 
+let frameworkPrefix = "no-framework"
```

Comment:
I would prefer to have this at the closest place where it is used, which is line 91